android/app/src/main/java/com/mattermost/helpers/database_extension/General.kt

```diff
@@ -57,7 +57,7 @@ fun DatabaseHelper.getDatabaseForServer(context: Context?, serverUrl: String): W
             if (cursor.count == 1) {
                 cursor.moveToFirst()
                 val databasePath = String.format("file://%s", cursor.getString(0))
-                return WMDatabase.getInstance(databasePath, context!!)
+                return WMDatabase.buildDatabase(databasePath, context!!, SQLiteDatabase.CREATE_IF_NECESSARY)
```

Comment:
this is to ensure we create a new database connection as when they arrive in quick succession the code complains that we are trying to open an already closed database.